src/alto-ui/Aside/Aside.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { bemClass } from '../helpers/bem';
+
+import CloseButton from '../CloseButton';
+
+import './Aside.scss';
+
+function Aside({ className, onClose, show, title, children, a11yCloseLabel }) {
+  if (!show) return null;
+  return (
+    <aside className={bemClass('Aside', {}, className)}>
+      <div className="Aside__header">
+        <div className="Aside__title">{title}</div>
+        {onClose && <CloseButton open={show} onClose={onClose} a11yLabel={a11yCloseLabel} />}
+      </div>
+      <div className="Aside__content">{children}</div>
+    </aside>
+  );
+}
+
+Aside.displayName = 'Aside';
+
+Aside.defaultProps = {
+  show: true,
```

Comment:
false by default no?

Reply:
nope: as it is optionnal (default props, not isRequired) then you can omit it but you want to display the panel anyway. Why ? perhaps you are doing the visibility yourself:

```jsx
{displayPanel && <Aside title="my panel">hello</Aside>}
```